src/main/webapp/resources/js/pages/announcement/components/AnnouncementsPage.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import {
+  getAnnouncements,
+  markAnnouncementRead,
+} from "../../../apis/announcements/announcements";
+import { Avatar, Form, List, notification, Radio, Typography } from "antd";
+import { PriorityFlag } from "./PriorityFlag";
+import { fromNow } from "../../../utilities/date-utilities";
+import ViewReadAnnouncement from "./ViewReadAnnouncement";
+import ViewUnreadAnnouncement from "./ViewUnreadAnnouncement";
+import { blue1 } from "../../../styles/colors";
+
+const { Text } = Typography;
```

Comment:
`Text` is never used

Reply:
Please see f9b969f01ca6ae0beeabde7bf7088990fba138d2.